test/junit/scala/util/ChainingOpsTest.scala

```diff
@@ -23,12 +23,10 @@ class ChainingOpsTest extends RunTesting {
 
   @Test
   def testAnyPipe: Unit = {
-    val times6 = (_: Int) * 6
-    val result = (1 - 2 - 3)
-      .pipe(times6)
-      .pipe(scala.math.abs)
+    val plus3 = (_: Int) + 3
+    val result = (1 - 2) |> plus3 |> scala.math.abs
```

Comment:
What if the user doesn't like postfix syntax ? I think `pipe()` should be there for us who do not like ascii art in our code.

Reply:
How will it look on multiple lines?

```scala
    val result = (1 - 2 - 3)
      |> times6
      |> scala.math.abs
```

Does that parse ok? Or would it need surrounding parentheses?

If not, I'm not sure these:

```scala
    val result = ((1 - 2 - 3)
      |> times6
      |> scala.math.abs)

    val result = (1 - 2 - 3)
      .|>(times6)
      .|>(scala.math.abs)
```

are nicer than with `.pipe`

Reply:
> What if the user doesn't like postfix syntax ?

@martin-g eh? there isn't any postfix syntax here

Reply:
I'm guessing you meant infix, rather than postfix.

Reply:
Unfortunately it does not parse, but like in the other languages, I think it should. But that shall be handled as a separate matter.

Reply:
I would too, if it compiled :-)

Reply:
```scala
Welcome to Scala 2.13.0-20181012-195043-19fa66d (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_161).
Type in expressions for evaluation. Or try :help.

scala> import scala.util.chaining._
import scala.util.chaining._

scala> def times6(i: Int) = i*6
times6: (i: Int)Int

scala> val result = (
     |   (1 - 2 - 3)
     |     |> times6
     |     |> scala.math.abs
     | )
result: Int = 24
```

it does! :-)

Reply:
Wow, did the brackets do the trick? Excellent. Will change.